src/Controller/AuthenticationController.php

```diff
@@ -388,16 +376,14 @@ private function showUserIsBlockedErrorPage(bool $isBlockedPermanently): Respons
         }
         // Forward to the exception controller to prevent an error being logged.
         return $this->forward(
-            'App\Controller\ExceptionController::showAction',
+            'App\Controller\ExceptionController::show',
```

Comment:
Is this namespace correct? I thought you changed it in composer to Surfnet\Tiqr?

Reply:
Good catch. Updated this on another place too.